osmoutils/accum/prefix.go

```diff
@@ -0,0 +1,24 @@
+package accum
+
+import "fmt"
+
+// formatAccimPrefix returns the key prefix used for any
+// accum module values to be stored in the KVStore.
+// Returns "accum/{key}" as bytes.
+func formatModulePrefixKey(key string) []byte {
+	return []byte(fmt.Sprintf("accum/%s", key))
+}
+
+// formatAccimPrefix returns the key prefix used
+// specifically for accumulator values in the KVStore.
+// Returns "accum/acc/{name}" as bytes.
+func formatAccumPrefixKey(name string) []byte {
```

Comment:
Note: while we only discussed prefixing positions, I think it is also useful to prefix the accumulators themselves in case we ever want to iterate over accumulators